cmd/fetchbranches/main.go

```diff
@@ -99,9 +99,9 @@ func fetchBranches(fetchMap map[string]string, pr pr) error {
 		fetchDirectory := filepath.Join(fullDirectory, pr.owner, branchRepo)
 		os.Chdir(fetchDirectory)
 
-		cmd = exec.Command("git", "fetch", "origin")
-		nameFetch := filepath.Join("origin", branchName)
-		cmd = exec.Command("git", "checkout", "-b", branchName, nameFetch)
+		// Brings the change of the PR
+		nameFetch := fmt.Sprintf("pull/%d/head", pr.number)
```

Comment:
you have to use the url defined in the tag

to pull the branch of a user you could use
```
 git pull https://github.com/$USER/$REPO $BRANCH  --no-edit
```

for example
```
git pull https://github.com/GabyCT/tests topic/fixfetch  --no-edit
```